src/Passport.php

```diff
@@ -691,4 +698,25 @@ public static function withoutCookieSerialization()
 
         return new static;
     }
+
+    /**
+     * Determine if refresh tokens must be revoked.
+     *
+     * @return bool
+     */
+    public static function revokeRefreshTokens()
+    {
+        return static::$revokeRefreshTokens;
+    }
```

Comment:
This method isn't needed. Just check the property directly. I realize this is inconsistent with the other methods in this class but they're also unneeded actually. We can remove those in a future release.